torch_geometric/nn/models/dimenet.py

```diff
@@ -242,11 +334,23 @@ def reset_parameters(self):
     def forward(self, x, rbf, i, num_nodes=None):
         x = self.lin_rbf(rbf) * x
         x = scatter(x, i, dim=0, dim_size=num_nodes)
+        x = self.lin_up(x)
         for lin in self.lins:
             x = self.act(lin(x))
         return self.lin(x)
 
 
+class OutputBlock(OutputPPBlock):
+    def __init__(self, num_radial, hidden_channels, out_channels, num_layers,
+                 act=swish):
+        super().__init__(num_radial, hidden_channels, hidden_channels,
+                         out_channels, num_layers, act=swish)
+        import warnings
+        warnings.warn(
+            "OutputBlock will be depreciated in PyG 2.1.0. Use OutputPPBlock instead",  # noqa
```

Comment:
Any reason we don't use `OutputPPBlock` this will be depreciated?

Reply:
The DimeNet++ paper says that OutputPPBlock has less information bottlenecks as it allows for information up projection and down projection of embedings. The difference between them is minor and I thought that OutputPPBlock will also benefit DimeNet model users. Hence the change.

But I have to test whether OutputPPBlock will be compatible with DimeNet when using pre-trained weights. If it is not compatible, I will revert back.

Reply:
Thank you. On that note, it might makes sense to add a test for loading pre-trained weights in general. Can try to do this in a separate PR.

Reply:
@Padarn I have restored OutputBlock as OutputPPBlock was not compatible with DimeNet when using pretrained weights.

@rusty1s , I would like to add tests in a separate PR.

Reply:
Thanks for the update @arunppsg. Once this is merged could you make an issue describing the `OutputPPBlock` problem? 🙇

Reply:
I don't think it's a problem - earlier DimeNet was using OutputBlock, I thought it can use OutputPPBlock since it is more efficient but realized that it will not be compatible, then reverted back. So, users who want to use OutputPPBlock can directly use DimeNet++

Reply:
Oh I see now, sorry I got mixed up because I was reading it via email. Thanks.